vpr/src/base/read_options.cpp

```diff
@@ -1754,6 +1785,97 @@ argparse::ArgumentParser create_arg_parser(std::string prog_name, t_options& arg
         .default_value("false")
         .show_in(argparse::ShowIn::HELP_ONLY);
 
+    place_grp.add_argument(args.place_static_move_prob, "--place_static_move_prob")
+        .help(
+            "The Probability of different move in Simulated Annealing."
+            "The order of moves is: {uniformMove, MedianMove, CentroidMove, WeightedCentroid, WeightedMedian, Timing feasible Region(TFR), Critical UniformMove}")
+        .nargs('+')
+        .default_value({"100", "0", "0", "0", "0", "0", "0"})
+
+        .show_in(argparse::ShowIn::HELP_ONLY);
+
+    place_grp.add_argument(args.place_static_notiming_move_prob, "--place_static_notiming_move_prob")
+        .help(
+            "The Probability of different non timing move in Simulated Annealing."
+            "The order of moves is: {uniformMove, MedianMove, CentroidMove}")
+        .nargs('+')
+        .default_value({"100", "0", "0"})
+
+        .show_in(argparse::ShowIn::HELP_ONLY);
+
+    place_grp.add_argument(args.place_high_fanout_net, "--place_high_fanout_net")
+        .help(
+            "Sets the assumed high fanout net during placement. ")
+        .default_value("10")
+        .show_in(argparse::ShowIn::HELP_ONLY);
+
+    place_grp.add_argument<bool, ParseOnOff>(args.simpleRL_agent_placement, "--simpleRL_agent_placement")
+        .help(
+            "Uses a simple RL agent in choosing the appropiate move type in placement. ")
```

Comment:
What is a simple RL agent?  Is this the opposite of multistate?  If so, why not just have one option (--place_agent_multistate on|off)?  If it does something else, you should define it here.

Reply:
This option just activates the RL placement instead of using a fixed probability for each move type. I have changed the option name to be RL_agent_placement and updates the help to be more clear.